jobs/build-arch.Jenkinsfile

```diff
@@ -603,8 +594,7 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
             }
             parallelruns['OSTree Import: Compose Repo'] = {
                 shwrap("""
-                cosa shell -- \
```

Comment:
can we put the removal of `cosa shell --` here and the `sync` of the `send-ostree-import-request.py` in a separate initial commit here. I think the subtlety there is worth it's own descriptive commit message.